content/organization/tech/security/index.en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Security"
+menuTitle: "Security"
+weight: 54
+---
+
+As organizers you will be responsible for accounts such as email, social media accounts.
+Here are some steps that will help reduce the risk of someone's hacking into your account.
+
+## Use a personal password manager!
+
+For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or a paid service like 1Password.
+
+If you feel on the fence, ask other organizers or your friends about their experience with some password managers.
+
```

Comment:
### Common password manager workflow

Password managers make it safe to store all your passwords in encrypted, safe storage. 
You only need to remember the password to unlock your password manager, to get access to all your account passwords.
This means this "master password" needs to be safe, meaning:
- Be at least several words long (using phrases from shows/songs you like ight be a good way to go)
- Contain special characters and numbers
- Be hard to guess, but easy for you to remember
- Not used in _any other instance_.